test/scripts/start-services.sh

```diff
@@ -41,11 +41,31 @@ start_parachain()
     pushd ../parachain
 
     cargo build
-    target/debug/artemis-node --tmp --dev >$logfile 2>&1 &
+
+    echo "Generating Parachain spec"
+    target/debug/artemis-node build-spec --dev --disable-default-bootnode > $configdir/spec.json
+
+    echo "Inserting Ganache chain info into genesis spec"
+    ethereum_initial_header=`curl http://localhost:8545 \
+        -X POST \
+        -H "Content-Type: application/json" \
+        -d '{"jsonrpc":"2.0","method":"eth_getBlockByNumber","params": ["latest", false],"id":1}' \
+        | node ../test/scripts/helpers/transformEthHeader.js`
+    node ../test/scripts/helpers/overrideParachainSpec.js $configdir/spec.json \
+        genesis.runtime.verifierLightclient.initialDifficulty 0x0 \
+        genesis.runtime.verifierLightclient.initialHeader "$ethereum_initial_header"
+
+    target/debug/artemis-node -lruntime=debug \
+        --alice \
```

Comment:
Adding `--alice` will probably break things as it will run the chain in PoA mode. From my vague recollection PoA mode requires at least two PoA validators to be running in the network. Can we remove this flag?

Reply:
Without this flag, the Substrate node doesn't produce blocks / process transactions in its transaction pool. I initially didn't have `--alice` and then realized the `--dev` flag isn't equivalent to using a chain spec with type "Development". Lots of digging later, and `--alice` is the easiest way to replicate `--dev` behaviour. Have a look at https://github.com/paritytech/substrate/blob/master/client/cli/src/commands/run_cmd.rs. I think `--dev` also runs in PoA mode with "forced authoring" of blocks

Reply:
Ahh Good to know 👍

Reply:
You're right. Tests are broken... back to the drawing board

Reply:
It was missing the `--force-authoring` flag. Now it's fairly equivalent to `--dev`. Tests pass